hooks/post/30-upgrade.rb

```diff
@@ -51,7 +51,7 @@ def import_puppet_modules
 end
 
 def import_subscriptions
```

Comment:
I don't think we can safely remove these. I think you are expecting users to upgrade to every successive version on their way along the way (e.g. 3.2 -> 3.3 ->3.4) however, we have no requirement on that and have users that will go 3.2 -> 3.4.